content/en/agent/vector_aggregation.md

```diff
@@ -0,0 +1,157 @@
+
+**Note**: Vector can also directly collect logs from alternative sources. If doing so, it is likely that 
+third party logs won't come with proper tagging. A convenient way to [add tags][8], source or service
+values is to use the [Vector Remap Language][3].
+
+## Configuration
+
+### Agent configuration
+To send logs to Vector, update the Agent configuration file, `datadog.yaml`.
+Only the logs data type is currently supported. Update the following values in the `datadog.yaml` file:
+```yaml
+logs_config:
+  logs_dd_url: "<VECTOR_HOST>:<VECTOR_PORT>"
+  logs_no_ssl: true # If TLS/SSL is not enabled on the vector side
+```
+
+### Vector configuration
+To receive logs from Datadog Agent, configure Vector with a [datadog_logs source][9].
+To send logs to Datadog, Vector should be configured with at least one [datadog_logs sink][10].
+
+See the official [Vector documentation][11] for all available configuration parameters and
+all transforms that can be applied to logs while they are processed by Vector.
+
+Here is a configuration example that adds a tag to every log using the Vector Remap Language:
+```yaml
+sources:
+  datadog_agents:
+    type: datadog_logs
+    address: "[::]:8080"
+
+transforms:
+  add_tags:
+    type: remap
+    inputs:
+      - datadog_agents
+    source: |
+      .ddtags = .ddtags + ",sender:vector"
+
+sinks:
+  to_datadog:
+    type: datadog_logs
+    inputs:
+       - add_tags
+    default_api_key: "${DATADOG_API_KEY_ENV_VAR}"
+    compression: gzip
+    encoding:
+      codec: json
+    healthcheck:
+      enabled: true
+```
+
+### Using Kubernetes
+
+Using the official Datadog chart the following values need to be updated:
+
+```yaml
+agents:
+  useConfigMap: true
+  agents.customAgentConfig:
+    logs_dd_url: "<VECTOR_HOST>:<VECTOR_PORT>"
+    logs_no_ssl: true # If TLS/SSL is not enabled on the vector side
+```
+
+For additional details about the Datadog Helm chart, see [the Kubernetes documentation][12].
+
+Vector provides an [official chart for aggregating data][13] that comes with a Datadog
+logs source preconfigured. For more information about installing Vector using Helm,
+see to the [official Vector documentation][14].
+
+To ultimately sent logs to Datadog, a `datadog_logs` sink need to be added to the Vector
+configuration. Vector's chart supports adding additional sinks in the `values.yaml` file.
+
+A sample `datadog_logs` sink should ideally include a `remap` transform
+that adds a Datadog tag to recognize logs forwarded by Vector:
+
+```yaml
+transforms:
+  add_tags:
+    type: remap
+    inputs:
+      - datadog_logs # It's the name of the preconfigured Datadog source
+    source: |
+      .ddtags = .ddtags + ",sender:vector"
+
+sinks:
+  to_datadog:
+    type: datadog_logs
+    inputs:
+       - add_tags
+    default_api_key: "${DATADOG_API_KEY_ENV_VAR}"
+    compression: gzip
+    encoding:
+      codec: json
+    healthcheck:
+      enabled: true
+```
```

Comment:
There seems to be a lot of duplication here with the previous chapter. Are we sure we want to list this config again?

Reply:
Good idea, I leveraged timberio/vector#8168 that provide some shortcut to say just use the same config in `customConfig` and that's it.